### DISCUSSION TITLE
Splitting and storing a single k32 plot into multiple drives

### DISCUSSION THREAD

**Discussion by cheezburga** [Chia-Network/chia-blockchain#7829] [Ideas]
I have many drives that are filled with k32 plots but each each drive has 70GB space left.  I know k31 plots are no longer in used for farming so the idea of using them to fill up the remaining 70GB is not feasible.  Is it possible for DEV to create a way to split a k32 plot into N parts and be stored in N number of drives using some kind of file naming convention.  E.g.  plot-k32-3032-07-27-18-21-16-abcdef.plot can be split into like 5 parts and each file name will be like 

plot-k32-3032-07-27-18-21-16-abcdef.plot_part_1_out_of_5
plot-k32-3032-07-27-18-21-16-abcdef.plot_part_2_out_of_5
plot-k32-3032-07-27-18-21-16-abcdef.plot_part_3_out_of_5
plot-k32-3032-07-27-18-21-16-abcdef.plot_part_4_out_of_5
plot-k32-3032-07-27-18-21-16-abcdef.plot_part_5_out_of_5

and each file can be stored anywhere in the list of plot directories.  When each Challenge going through the filter, it would attempt to find 1st part "plot-k32-3032-07-27-18-21-16-abcdef.plot_part_1" and then search for remaining parts to process the challenge as if were process in the original plot "k32-3032-07-27-18-21-16-abcdef.plot".  Of course if some parts went missing, then this plot cannot be used for challenge.

**Comment by mariano54** [comment]
It's theoretically possible, but you would have to make some changes to chiapos.
Much easier would be to RAID your drives somehow

**Reply by cheezburga** [reply]
Thanks for the quick response.  I just thought it would be a cool feature for the average joe users that primary use windows and plug and play usb drives to leverage every GB in their drives without touching advance feature like RAID.  I guess in the meantime I will do some research on RAIDing my usb drives lol.

**Comment by keliew** [comment]
You could swap out the K32 and replace with _some_ K33 or K34.

**Reply by cheezburga** [reply]
I am using 15k 300GB SAS drive for plotting with madmax at the moment.  K33 and K34 might not fully occupy the remaining GB.  I am going to try splitting the plot file and use affuse to mount the pieces of files as a virtual file in ubuntu.  I will see if that works later.

**Comment by joelrb** [comment]
If you're using Windows, somebody posted a guide about this in the Chia forum.

https://chiaforum.com/t/tip-for-using-that-last-bit-of-space-on-your-drives/6400

I urged you though to read all the comments. Somebody accidentally reformatted all his drives following the guide. So be careful.

**Reply by cheezburga** [reply]
Thanks for sharing, I have read it too and yes I also read someone reformatted by accident lol.  I am not using windows because it is slow for plotting.  I am looking for an elegant solution without having the risk to replot when I move my usb drive between systems.  RAID0 sounds good for short term but when one drive fail over the course of next 5-10 year I will have to replot.  I plan on selling the plotter once everything are filled up to recoup the cost.
